### PR TITLE
prepare 1.24.0

### PR DESCRIPTION
**Pull request by r10s**
already preparing things, the release will take some more days as there are open points wrt [api30](https://github.com/deltachat/deltachat-android/issues/1849)

based on [core62](https://github.com/deltachat/deltachat-core-rust/pull/2769), corresponding ios pr at https://github.com/deltachat/deltachat-ios/pull/1383

this is how the device message looks like currently:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
+* fix: disappearing messages timer now synced more reliable in groups
+* fix: improve detection of some mailing list names
+* fix "QR process failed" error
+* fix DNS and certificate issues
+* fix: if account creation was aborted, go to the previously selected account, not to the first
+* fix back button not working in connectivity view sometimes
+* fix: disable chat editing options if oneself is not a member of the group
+* fix shared image being set as draft repeatedly
+* fix: hide keyboard when compose panel is hidden
+* fix "jump to section" links in local help
```
Comment (adbenitez) [inline CHANGELOG.md:34]: I think it is not about the local help, that was working in the past since they are local files, the issue was with HTML emails

Comment (r10s) [reply CHANGELOG.md]: you're right, in local help, these links work as expected. i changed that, thanks a lot for reading carefully :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,44 @@
 # Delta Chat Android Changelog
 
+## v1.24.0
+2021-10
```
Comment (adbenitez) [inline CHANGELOG.md:4]:
```suggestion
2021-11
```

Comment (r10s) [reply CHANGELOG.md]: time passes :)

**Review by adbenitez — APPROVED** [review]

**Pull request merged**